tests/network/upgrade/conftest.py

```
@@ -381,3 +389,111 @@ def ipv4_dedicated_nic_bridge_localnet_address_pool_upgrade() -> Generator[IPv4I
+        )
+        yield udn
+
+
+UDN_UPGRADE_VM_ANTI_AFFINITY_LABEL: Final[dict[str, str]] = {"upgrade-udn-vm": "true"}
+
+
+@pytest.fixture(scope="session")
+def udn_upgrade_unprivileged_user_role_binding(
+    admin_client: DynamicClient,
+    udn_upgrade_namespace: Namespace,
+    udn_upgrade_layer2: Layer2UserDefinedNetwork,
```

Ruff (0.16.2)

[warning] Unused function argument: `udn_upgrade_layer2`

(ARG001)